analytic_operating_unit/models/account_analytic_account.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2016-17 Eficent Business and IT Consulting Services S.L.
+# © 2016-17 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import fields, models
```

Comment:
From my point of view the travis fail is depending on this. Before, it was importing from openerp and all checks were green and runbot was fine. The same with https://github.com/OCA/operating-unit/pull/47 and https://github.com/OCA/operating-unit/pull/48